content/vi/_index.html

```diff
@@ -41,12 +41,12 @@ <h2>Những thách thức của việc di chuyển hơn 150 Microservices lên K
         <button id="desktopShowVideoButton" onclick="kub.showVideo()">Xem Video</button>
         <br>
         <br>
-        <a href="https://events.linuxfoundation.org/events/kubecon-cloudnativecon-europe-2020/" button id="desktopKCButton">Tham gia KubeCon Amsterdam 30/03-02/04/2020</a>
+        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america" button id="desktopKCButton">Tham gia KubeCon Bắc Mỹ 24-28/10/2022</a>
         <br>
         <br>
         <br>
         <br>
-        <a href="https://events.linuxfoundation.cn/kubecon-cloudnativecon-open-source-summit-china/" button id="desktopKCButton">Tham gia KubeCon Thượng Hải 28-30/7/2020</a>
+        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe-2023/?utm_source=kubernetes.io&utm_medium=nav&utm_campaign=kccnceu23" button id="desktopKCButton">Tham gia KubeCon Châu Âu 17-21/04/2023</a>
```

Comment:
Please update the url and the time.
The correct one is: 
- URL: https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/
- Time: 18-21/04/2023